### ISSUE TITLE
Observed Fairness was not dropped for baseline model

### ISSUE THREAD

**Issue by dileeppanigrahi** [princetonvisualai/gan-debiasing#10]
Hi ,

Baseline model fairness was not dropped for the Male is protected attribute and Smiling is the target attribute. 

I've trained the standard attribute classifier using the below command  `python main.py --experiment baseline`  
As per the code, The baseline model is trained on the

`Dataset:`  CelebA training dataset X with 162,770 images,
`Hyperparameters:`  Binary cross-entropy loss for 20 epochs with a batch size of 32, and using Adam optimizer with a learning rate of 1e-4.
`Target attribute: Smilling (attribute 31)`
`Protected attribute: Male (attribute 20)`

Fairness metrics for Baseline model : 

> Training epoch 19: [5001|5087], loss:0.07021914422512054
> Avg precision all = 0.9827746330855125
> Validation results:
> AP : {:.1f} +- {:.1f} 98.52633278341692 0.13046754108938044
> DEO : {:.1f} +- {:.1f} 3.3444695945977543 1.1048136540690787
> BA : {:.1f} +- {:.1f} 0.06943005310537514 0.3600587245308783
> KL : {:.1f} +- {:.1f} 0.01865371993659408 0.03730743987318816
> Test results:
> AP : {:.1f} +- {:.1f} 98.43127724632386 0.1274417440206799
> DEO : {:.1f} +- {:.1f} 2.7732756124918376 1.1788377126697231
> BA : {:.1f} +- {:.1f} -0.5825026659862016 0.4052127065723986
> KL : {:.1f} +- {:.1f} 0.020252475423118695 0.04050495084623739

if you observe the above fairness matrics (AP, DEO, BA, KL) looks fine for the baseline model, 
but the result is shown in the paper for the baseline model is different  (DEO, BA, KL is high).

As per my understanding the standard classifier fairness metrics:  DEO, BA, KL should be high.

do I need to change anything to train the standard attribute classifier to reproduce the paper results(fairness)?

Please let me know whether I'm doing any mistakes. 
Expecting your response.

**Comment by dileeppanigrahi** [comment]
@vramaswamy94, @sunniesuhyoung can you please respond. Thank you

**Comment by ozgurkara99** [comment]
Hi,

I have also tried the reproduce the results as @dileeppanigrahi did, but I also got similar fairness results for baseline (for smiling classifier when "male" is protected attribute) to what @dileeppanigrahi found. How can I obtain comparatively high fairness scores as reported in the paper? Could you please help?

Thank you

**Comment by vramaswamy94** [comment]
`Smiling` as an attribute is not that biased, and the baseline fairness scores are similar to the ones we got. Attributes like `ArchedEyebrows`, `BlondHair`, etc. have higher bias.
